tree/10_ncf_internals/modules/promises/getParent.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+import os
+import sys
+import glob
+
+def get_parent(path):
+  return os.path.dirname(os.path.realpath(os.path.normpath(path)))
+
+def get_parents(path, parent_dirs):
+  current_dir = os.path.realpath(path)
+  if (os.path.isfile(path)):
```

Comment:
why not use "if then else" here instead of "define + if then override"
it xould be more readable